backend/src/constants/env.js

```diff
@@ -18,7 +35,24 @@ if (error) {
 const env = {
   NODE_ENV: envVars.NODE_ENV,
   PORT: envVars.PORT,
-  MONGODB_URL: envVars.MONGODB_URL,
+
+  DB_USER_production: envVars.DB_USER_PROD,
+  DB_PASS_production: envVars.DB_PASS_PROD,
+  DB_HOST_production: envVars.DB_HOST_PROD,
+  DB_PORT_production: envVars.DB_PORT_PROD,
+  DB_NAME_production: envVars.DB_NAME_PROD,
+
+  DB_USER_development: envVars.DB_USER_DEV,
+  DB_PASS_development: envVars.DB_PASS_DEV,
+  DB_HOST_development: envVars.DB_HOST_DEV,
+  DB_PORT_development: envVars.DB_PORT_DEV,
+  DB_NAME_development: envVars.DB_NAME_DEV,
+
+  DB_USER_test: envVars.DB_USER_TEST,
+  DB_PASS_test: envVars.DB_PASS_TEST,
+  DB_HOST_test: envVars.DB_HOST_TEST,
+  DB_PORT_test: envVars.DB_PORT_TEST,
+  DB_NAME_test: envVars.DB_NAME_TEST,
```

Comment:
mixing cases in this scenario may be confusing it is safer to keep eg. BD_USER_PROD: env.Vars.DB_USER_PROD and just rename NODE_ENV valid variables from 'production', 'development', 'test' to "PROD", "DEV", "TEST"